### PR TITLE
Correct path separator in Windows Build instructions

### PR DESCRIPTION
**Pull request by batjko**
Just a little oversight.

### CODE REVIEW

**Comment by mnquintana** [comment]
Nice catch - thanks!

**Comment by batjko** [comment]
:+1:

File: docs/build-instructions/windows.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ cd atom
 script/build # Creates application in the `Program Files` directory
```
Comment (izuzak) [inline docs/build-instructions/windows.md]: I noticed you didn't change this instance of `script/build`. Just wondering why that's so.

**Comment by izuzak** [comment]
I can't run my windows VM anymore, so I can't try this out, but I think it was possible to run `script/build` in Git shell. Can anyone give give that a try?

**Comment by mnquintana** [comment]
@izuzak Ah that's a good point, that should work. Should we revert this?

**Comment by batjko** [comment]
I can try it via the git shell... will update shortly.

I wouldn't revert the change but clarification may be necessary, because in command or powershell it still needs to be a backslash. So if it does depend on from where the build is executed, we may have to mention that.

File: docs/build-instructions/windows.md (hunk 1 above)
Comment (batjko) [inline docs/build-instructions/windows.md]: My oversight in this instance. 
I'll open a new PR to correct this, as soon as we are also clear about whether further clarification needs to be added.

**Comment by batjko** [comment]
Confirmed. The shell does take fwd slashes.
So I have added #8109 to partially revert this one and add a note to mention backslashes if the shell isn't used.

**Pull request merged**